packages/core/admin/webpack.config.js

```diff
@@ -96,41 +79,79 @@ module.exports = ({
       rules: [
         {
           test: /\.m?js$/,
-          // TODO remove when plugins are built separately
-          include: [cacheDir, ...pluginsPath],
+          include: cacheDir,
+          oneOf: [
+            // Use babel-loader for files that distinct the ee and ce code
+            // These files have an import Something from 'ee_else_ce/
+            {
+              test(filePath) {
+                if (!filePath) {
+                  return false;
+                }
+
+                try {
+                  const fileContent = fse.readFileSync(filePath).toString();
+
+                  if (fileContent.match(/from.* ['"]ee_else_ce\//)) {
+                    return true;
+                  }
+
+                  return false;
```

Comment:
What do you think about defining the regex before, so that it doesn't need to be compiled for each file and to change this implementation a little to:

```
return `/from.* ['"]ee_else_ce\//.test(fileContent)`
```

or

```
const EE_REGEX = /from.* ['"]ee_else_ce\//;

// ...

return `EE_REGEX.test(fileContent)`
```

This way we don't have to allocate the result, which saves about 30 - 40MB of memory on my machine.

I tried to check whether `fs-extra` does have a bigger memory footprint than `fs` but couldn't find anything conclusive, so I just want to note, that I think `fse` is not needed here and it could be just plain `fs`.

Reply:
`fse` is already installed in the package's dependencies so I think it's better to stick with it as it is also used in the BE.